datastore/schedule-export/src/main/java/com/example/datastore/HelloAppEngine.java

```diff
@@ -0,0 +1,33 @@
+package com.google.example.datastore;
+
+import com.google.appengine.api.utils.SystemProperty;
+
+import java.io.IOException;
+import java.util.Properties;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet(name = "HelloAppEngine", value = "/hello")
+public class HelloAppEngine extends HttpServlet {
```

Comment:
This should also probably be named a Servlet.

Reply:
Deleted the HelloAppEngine servlet.